vault/src/main/java/org/opencadc/vault/VaultInitDatabase.java

```diff
@@ -0,0 +1,106 @@
+*  products derived from this           promouvoir les produits dérivés
+*  software without specific prior      de ce logiciel sans autorisation
+*  written permission.                  préalable et particulière
+*                                       par écrit.
+*
+*  This file is part of the             Ce fichier fait partie du projet
+*  OpenCADC project.                    OpenCADC.
+*
+*  OpenCADC is free software:           OpenCADC est un logiciel libre ;
+*  you can redistribute it and/or       vous pouvez le redistribuer ou le
+*  modify it under the terms of         modifier suivant les termes de
+*  the GNU Affero General Public        la “GNU Affero General Public
+*  License as published by the          License” telle que publiée
+*  Free Software Foundation,            par la Free Software Foundation
+*  either version 3 of the              : soit la version 3 de cette
+*  License, or (at your option)         licence, soit (à votre gré)
+*  any later version.                   toute version ultérieure.
+*
+*  OpenCADC is distributed in the       OpenCADC est distribué
+*  hope that it will be useful,         dans l’espoir qu’il vous
+*  but WITHOUT ANY WARRANTY;            sera utile, mais SANS AUCUNE
+*  without even the implied             GARANTIE : sans même la garantie
+*  warranty of MERCHANTABILITY          implicite de COMMERCIALISABILITÉ
+*  or FITNESS FOR A PARTICULAR          ni d’ADÉQUATION À UN OBJECTIF
+*  PURPOSE.  See the GNU Affero         PARTICULIER. Consultez la Licence
+*  General Public License for           Générale Publique GNU Affero
+*  more details.                        pour plus de détails.
+*
+*  You should have received             Vous devriez avoir reçu une
+*  a copy of the GNU Affero             copie de la Licence Générale
+*  General Public License along         Publique GNU Affero avec
+*  with OpenCADC.  If not, see          OpenCADC ; si ce n’est
+*  <http://www.gnu.org/licenses/>.      pas le cas, consultez :
+*                                       <http://www.gnu.org/licenses/>.
+*
+************************************************************************
+*/
+
+package org.opencadc.vault;
+
+import java.net.URL;
+import java.util.Arrays;
+import javax.sql.DataSource;
+import org.apache.log4j.Logger;
+import ca.nrc.cadc.db.version.InitDatabase;
+
+/**
+ *
+ * @author pdowler
+ */
+public class VaultInitDatabase extends InitDatabase {
+    private static final Logger log = Logger.getLogger(VaultInitDatabase.class);
+
+    static String[] CREATE_SQL = new String[] {
+        "vospace.ModelVersion.sql",
+        "vospace.NodeAvailability.sql",
+        "vospace.Node.sql",
+        "vospace.NodeProperty.sql",
+        "vospace.DeletedNode.sql"
+    };
+
+    public static final String MODEL_NAME = "vospace";
+    public static final String MODEL_VERSION = "0.1";
+    public static final String PREV_MODEL_VERSION = "0.1";
```

Comment:
PREV version should be "n/a". Def not null (that means something "any" and is only useful in special cases).

Reply:
Updated to n/a.